openach/tasks.py

```diff
@@ -15,3 +21,24 @@ def example_task(x, y):  # pragma: no cover
     An example for reference.
     """
     return x + y
+
+@shared_task
+def update_url_title_description(source_id):
+	source=EvidenceSource.objects.get(id = source_id)
+	try:	
+		html = requests.get(source.source_url)
+		doc = BeautifulSoup(html.text, "html.parser")
+		ogs = doc.html.head.findAll(property=re.compile(r'^og'))
+		url_info = {}
+		for og in ogs:
+			if og.has_attr('content'):
+				url_info[og['property'][3:]] = og['content']
+		if 'title' not in url_info:
+			url_info['title'] =  doc.title.text
+			url_info['description'] = ""
+		source.source_url_title = url_info['title']
+		source.source_url_description = url_info['description']
+		logger.info(url_info)
+		source.save()
+	except Exception as e:
+		logger.info(e)
```

Comment:
- Catching the general exception should generally be avoided. What are the different exceptions that we need to handle?
- For each exception, what should be done? For example, if we can't access the URL for some reason, do we need to re-try later? Would be interested in your thoughts